Docs/roles-and-permissions.md

```diff
@@ -0,0 +1,55 @@
+# Manually setting up roles and assigning permissions in Azure Active Directory (AAD)
```

Comment:
Let's update the [Guidance on customizing preconfigured solutions doc](https://azure.microsoft.com/en-us/documentation/articles/iot-suite-guidance-on-customizing-preconfigured-solutions/#manage-the-permissions-in-a-preconfigured-solution) with steps below.